recipe/meta.yaml

```
@@ -1,33 +1,44 @@
 {% set name = "libcst" %}
-{% set version = "0.3.19" %}
+{% set version = "1.6.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 4876239db55164acaf034ee01f56a7db0a2f90cacea24b183d8aa69efc11b067
+  sha256: e80ecdbe3fa43b3793cae8fa0b07a985bd9a693edbe6e9d076f5422ecadbf0db
 
 build:
   number: 0
-  skip: true  # [py < 36]
-  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [py<39]
+# Cargo is actively removed as the upstream package calls in a duplicate build, causing windows builds to fail.
+  script:
+    - rm %BUILD_PREFIX%\\.cargo.win\\config  # [win]
+    - {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
+  missing_dso_whitelist:  # [s390x]
+    - $RPATH/ld64.so.1    # [s390x]
+
 
 requirements:
-  host:
-    - dataclasses   # [py < 37]
-    - pip
+    build:
+    - setuptools
+    - setuptools-rust
+    - setuptools_scm
+    - wheel
+    - {{ compiler('rust') }}
+    - {{ compiler('cxx') }}
```

```suggestion
    - {{ compiler('cxx') }}
```
I couldn't find anything that said we needed this. Can you please help me understand where this comes from?

---

You're right! I think there was some confusion regarding if our rust needed this compiler and apparently we don't. I'll remove it thanks!

---

Haha sorry, been working on this for a few days and actually forgot that the compiler is used for linux support. I've added it back.